CMakeLists.txt

```diff
@@ -275,12 +275,17 @@ add_custom_target(
 )
 
 
+set(VIAMCPPSDK_ABSL_VERSION_MINIMUM)  # Someday?
```

Comment:
Abseil doesn't seem to set a useable semantic version, so there is no minimum we can declare. We declare an aspirational slot where a future value could go.